### ISSUE TITLE
Importance of eval function sanity check

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEATv2#391]
I really want to make sure that the evaluation functions we are evolving actually matter.

Make a DummayEvaluationFunction that always returns a value of 0.

Have MCTS, UCT, and Portfolio agents use this evaluation function to play games against the opponents you are using in your enemy sequence ... you should be able to easily do this using the Frontend, as long as you add the dummy function the the list of possible eval functions. Make these agents use the dummy eval function in multiple games (is there are option to run the game lots of times and average the results?) and log the results in the Google Drive directory.

Basically, we want to see how well these agents can perform with a terrible evaluation function. This will give is a baseline of performance. Once we do this, we can compare scores using our evolved evaluation function to the dummy evaluation function ... we need to make sure that the NNs we are evolving actually matter.

**Comment by alicequint** [comment]
portfolio ai isnt available in the front end by default so im going to try to add it because i think that, since were using it so often, that information could be very useful

**Comment by alicequint** [comment]
here are the results:
https://docs.google.com/document/d/13-LhgMWnu_hquIzPqAIZqQKjhhFQLHZESl6me_tW230/edit

**Comment by schrum2** [comment]
These results are very helpful, and indicate to me that we should stop evolving agents using the Portfolio agent. It is a good agent, but a lot of its skill comes from its human-designed components, so there doesn't seem to be much room for evolution to do anything interesting.

Focus on evolving evaluation functions for MLPSMCTS, UCT, and PuppetSearch.
Note: we eventually want to defeat these agents when they are using the sqrt3 evaluation function, so having these agents, and also the Portfolio agent, near the end of the enemy sequence (if we can ever reach it) would make sense.
